rasa/nlu/tokenizers/jieba_tokenizer.py

```diff
@@ -108,12 +111,21 @@ def preprocess_text(self, text, attribute):
         else:
             return text
 
-    def tokenize(self, text: Text, attribute=MESSAGE_TEXT_ATTRIBUTE) -> List[Token]:
+    def tokenize(
+        self, text: Text, attribute: Text = MESSAGE_TEXT_ATTRIBUTE
+    ) -> List[Token]:
         import jieba
 
         text = self.preprocess_text(text, attribute)
         tokenized = jieba.tokenize(text)
         tokens = [Token(word, start) for (word, start, end) in tokenized]
+
+        if (
```

Comment:
it seems to be the same in all tokenizers. Shall we move it to tokenizer superclass?

Reply:
except for spacy, but we can move it for the others